deployment/terraform/dump.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost-server/server/v8/platform/shared/mlog"
+)
+
+func (t *Terraform) IngestDump() error {
```

Comment:
Can we add some documentation here? Mainly what config is used and an overview of the commands it executes.

Reply:
done